ci-operator/step-registry/ipi/conf/networking/ipi-conf-networking-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG=/tmp/secret/install-config.yaml
+
+cluster_network_type=
+if [[ -e "${SHARED_DIR}/cluster-network-type.txt" ]]; then
+	cluster_network_type=$(<"${SHARED_DIR}/cluster-network-type.txt")
+fi
+
+cluster_variant=
+if [[ -e "${SHARED_DIR}/install-config-variant.txt" ]]; then
+	cluster_variant=$(<"${SHARED_DIR}/install-config-variant.txt")
+fi
+
+function has_variant() {
+	regex="(^|,)$1($|,)"
+	if [[ $cluster_variant =~ $regex ]]; then
+		return 0
+	fi
+	return 1
+}
+
+if has_variant "ovn"; then
+	cluster_network_type=OVNKubernetes
```

Comment:
@wking shouldn't this be `network_type`?

Reply:
I made it `cluster_network_type` when reading the variable in, to match `cluster_variant` (which is not just `variant`).  Fixed the consumers to be consistent in 055e091 -> df5d688.